js/privatebin.js

```diff
@@ -43,10 +43,21 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         var me = {};
 
         /**
+         * list of UserAgents (parts) known to belong to a bot
+         *
+         * @private
+         * @enum   {Object}
+         * @readonly
+         */
+        var BadBotUA = [
```

Comment:
I have not tested it actually. The strings here were merely templates, which could target many bots…
Maybe better remove it and wait for issues being opened, when one discovers such a bot…
Then we can add it to this blacklist.